strongbox-parent/pom.xml

```diff
@@ -65,7 +65,7 @@
         <version.groovy>2.4.7</version.groovy>
         <version.guava>19.0</version.guava>
         <version.hamcrest>1.3</version.hamcrest>
-        <version.httpclient>4.5</version.httpclient>
+        <version.httpclient>4.5.4</version.httpclient>
```

Comment:
It would be best to revert this to the old value, so that the validator part could be handled in this pull. We can handle the upgrade in a separate pull.